Dockerfile

```diff
@@ -58,9 +61,6 @@ RUN mv $BUILD_PATH/$PROJECT_BIN /bin/$PROJECT_BIN
 #
 FROM debian:buster AS default
 
-ARG PROJECT=akash
```

Comment:
I'm not sure we can remove these ARGs from this build step - what was the reason for these Dockerfile changes?

Reply:
Due to buildkit not being enabled for my builds earlier, I juggled around some arguments as they were not being passed to steps that needed them. 
Forgot to fix the order post enabling it.